### PR TITLE
NEP: accept NEP 20 partially (frozen, flexible, but not broadcastable dimensions)

### PR DESCRIPTION
**Pull request by mhvk**
As requested in #11423, a PR to change the status of NEP 20 to accepted.

### CODE REVIEW

**Comment by mattip** [comment]
Needs a `Resolution` tag with a link to the most relevant discussion threads

**Comment by mhvk** [comment]
OK, added `resolution` tag.

**Comment by shoyer** [comment]
Did we actually reach consensus here? If I recall, there may be still be some objections to the flexible/broadcastable parts.

**Comment by mattip** [comment]
I hope this PR stimulates a decision. If not maybe we need another ping to the mailing list.

My opinion is that if we do one of the options, the others cost virtually nothing in terms of runtime processing, and allow flexibility that are desirable, if not for NumPy then for other packages. As far as `matmul` goes, we can still reject using this mechanism for it while leaving fleible core dimensions for other, as yet unkown, gufuncs.

**Comment by mhvk** [comment]
I sent another message to the mailing list earlier today (as part of the thread on the NEP)

**Comment by mhvk** [comment]
Now changed to reflect what seems to be the emerging consensus on the mailing list: to accept flexible and frozen dimensions, which have clear use cases, and defer the broadcastable ones, which, as yet, do not.

File: doc/neps/nep-0020-gufunc-signature-enhancement.rst [hunk 1]
```diff
@@ -3,9 +3,12 @@ Expansion of Generalized Universal Function Signatures
 ======================================================
 
 :Author: Marten van Kerkwijk <mhvk@astro.utoronto.ca>
-:Status: Draft
+:Status: Accepted frozen and flexible, deferred broadcastable.
```
Comment (mattip) [inline doc/neps/nep-0020-gufunc-signature-enhancement.rst]: You cannot use arbitrary text here, the build fails

**Comment by mattip** [comment]
All the text referring to broadcast should be moved to an `Alternatives` section.

**Comment by mhvk** [comment]
Trying with a different formating, to see if tests pass.

**Comment by mhvk** [comment]
@mattip - Is that reshuffling really needed?

**Comment by mattip** [comment]
Missing a `Resolution` tag, should have the links to the email discussions, they can be on more than one lines as in
```
:Resolution:  url-link1,
url-link2
```

**Comment by mhvk** [comment]
It does have a `Resolution` tag (line 9). Am I missing something?

**Comment by mattip** [comment]
move the first link to the same line as the tag

**Comment by shoyer** [comment]
Thanks Martin!

**Pull request merged**